### PR TITLE
Version: 3.35.1

### PR DESCRIPTION
**Pull request by kira-syslogng**
Tarball is available at: https://github.com/syslog-ng/syslog-ng/actions/runs/1443669436

TODO:
- [x] #3770
- [x] #3809

### CODE REVIEW

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Comment by kira-syslogng** [comment]
Build SUCCESS

File: packaging/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+syslog-ng (3.35.1-1) unstable; urgency=medium
```
Comment (MrAnno) [inline packaging/debian/changelog:1]: I think we should change the `distribution` to `stable` as we use that name in the APT repo as well.

**Comment by bazsi** [comment]
Debian usually builds stuff in unstable and then those binaries become the
new stable. So it doesn't have to match. But I think it's better to keep it
this way so whenever the Debian stuff is synced, this won't be a difference.

**Comment by MrAnno** [comment]
Thanks, @bazsi.

**Review by Kokan — APPROVED** [review]

**Comment by kira-syslogng** [comment]
Build SUCCESS

**Review by MrAnno — APPROVED** [review]

**Pull request merged**